src/ReleaseApp/Domain/Entities/UpgradeInstructionMeta.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace ReleaseApp\Domain\Entities;
+
+use ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection;
+use SprykerSdk\Integrator\Common\UtilText\TextCaseHelper;
+
+class UpgradeInstructionMeta
+{
+    /**
+     * @var string
+     */
+    protected const INCLUDE_KEY = 'include';
+
+    /**
+     * @var string
+     */
+    protected const EXCLUDE_KEY = 'exclude';
+
+    /**
+     * @var string
+     */
+    protected const CONFLICT_KEY = 'conflict';
+
+    /**
+     * @var \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection|null
+     */
+    protected ?UpgradeInstructionModuleCollection $includeModuleCollection = null;
+
+    /**
+     * @var \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection|null
+     */
+    protected ?UpgradeInstructionModuleCollection $excludeModuleCollection = null;
+
+    /**
+     * @var \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection|null
+     */
+    protected ?UpgradeInstructionModuleCollection $conflictModuleCollection = null;
+
+    /**
+     * @var array<mixed>
+     */
+    protected array $body;
+
+    /**
+     * @param array<mixed> $body
+     */
+    public function __construct(array $body)
+    {
+        $this->body = $body;
+    }
+
+    /**
+     * @return \ReleaseApp\Domain\Entities\Collection\UpgradeInstructionModuleCollection
+     */
+    public function getInclude(): UpgradeInstructionModuleCollection
+    {
+        if ($this->includeModuleCollection) {
```

Comment:
It's an entity. Better initialize collections in constructor.

Reply:
++